pages/previewPage.html

```diff
@@ -276,10 +293,28 @@
+          />
+        </svg>
+      </button>
       <span id="percent"></span>
-      <button id="zoomFit" onclick="zoomFit()" title="Fit to window">
-        Fit to window
+      <button id="zoomFit" onclick="zoomFit()" title="Zoom to Fit">
+        <svg id="zoom-to-fit" viewBox="0 -25 122 140">
+          <title>zoom-to-fit</title>
+          <path
+            d="M86.64,12a6,6,0,1,1,0-12H113.8a6,6,0,0,1,6,6V33.66a6,6,0,1,1-12,0V20.45L84.12,44.23a6,6,0,0,1-8.47-8.42L99.41,12ZM12,36.24a6,6,0,1,1-12,0V9.08a6,6,0,0,1,6-6H33.66a6,6,0,1,1,0,12H20.45l23.77,23.7a6,6,0,1,1-8.41,8.47L12,23.47V36.24Zm24.27,64.92a6,6,0,0,1,0,12H9.08a6,6,0,0,1-6-6V79.47a6,6,0,1,1,12,0V92.68l23.7-23.77a6,6,0,0,1,8.47,8.41L23.47,101.16Zm74.67-24.27a6,6,0,0,1,12,0v27.16a6,6,0,0,1-6,6H89.22a6,6,0,1,1,0-12h13.21L78.65,74.37a6,6,0,0,1,8.42-8.47l23.84,23.76V76.89Z"
+          />
+        </svg>
+      </button>
+      <button id="refresh" onclick="refreshPage()" title="Refresh Page">
```

Comment:
Kind of nondescript, how about a more domain-specific wording like just `Recompile`